Emitron/Emitron/UI/Library/Filtering/AppliedFilterTagButton.swift

```diff
@@ -120,7 +120,7 @@ struct AppliedFilterView_Previews: PreviewProvider {
   
   static var tags: some View {
     HStack {
-      AppliedFilterTagButton(name: "Clear All", type: .destructive) { }
+      AppliedFilterTagButton(name: "Reset filters", type: .destructive) { }
```

Comment:
Again nit - filters should be capitalised

```suggestion
      AppliedFilterTagButton(name: "Reset Filters", type: .destructive) { }
```